### ISSUE TITLE
File management

### ISSUE THREAD

**Issue by RWOverdijk** [gitterHQ/gitter#1397]
It would be cool if there were some kind of file management.

It's already possible to upload and share files, but after that they seem to get lost in chat history.

**Comment by MadLittleMods** [comment]
Can you expand the type of managing you want to do?

**Comment by RWOverdijk** [comment]
Sure! It would be nice to have a button at the top right, that when clicked shows you the files you've uploaded (regardless of channel) and allows you to filter (for instance on channel). It doesn't have to be exactly like slack (which allows comments, sharing etc) but it would be nice to not upload the same file over and over again because you've lost the link. Like this, you can find the stuff you've uploaded.

![](https://i.ytimg.com/vi/duVAj8TqGRc/maxresdefault.jpg)

**Comment by prudhomm** [comment]
i agree, We share url (eg youtube) as well as fils (pdf, images, google docs ) in feelpp consortium. unfortunately there are currently no way to organize them in a forum and have easily access to all the shared files.

It would be very nice to have access to, at least, the list of the shared files in a forum. A way to organize them say in folders would be a big plus :)

**Comment by prudhomm** [comment]
what is the status on this one ?
